workflows/autometa-large-data-mode.sh

```diff
@@ -241,10 +241,10 @@ then
         set -x
```

Comment:
Would you mind adding a comment of an example header?

```suggestion
        # <insert example ORF header>
        set -x
```